src/ui/server/helpers/summary-lists/your-business/turnover-fields/index.test.ts

```diff
@@ -25,12 +25,15 @@ describe('server/helpers/summary-lists/your-business/turnover-fields', () => {
   const { referenceNumber } = mockApplication;
 
   const expectedBase = [
-    fieldGroupItem({
-      field: getFieldById(FIELDS.TURNOVER, ESTIMATED_ANNUAL_TURNOVER),
-      data: mockAnswers,
-      href: `${INSURANCE_ROOT}/${referenceNumber}${TURNOVER_CHANGE}#${ESTIMATED_ANNUAL_TURNOVER}-label`,
-      renderChangeLink: true,
-    }),
+    fieldGroupItem(
+      {
+        field: getFieldById(FIELDS.TURNOVER, ESTIMATED_ANNUAL_TURNOVER),
+        data: mockAnswers,
+        href: `${INSURANCE_ROOT}/${referenceNumber}${TURNOVER_CHANGE}#${ESTIMATED_ANNUAL_TURNOVER}-label`,
+        renderChangeLink: true,
+      },
+      `£${mockAnswers[ESTIMATED_ANNUAL_TURNOVER]}`,
```

Comment:
Should we/could we use `formatCurrency`?

Reply:
agreed to not use this here as we want to keep the user input shown